packages/frontend/src/components/userActious/gitcoin/SufficientUserScore.tsx

```diff
@@ -0,0 +1,21 @@
+import { FormHeading, FormRow, FormWrapper } from '@/components/form'
+import { Button } from '@/components/buttons'
+import { UserScoreProps } from '@/components/userActious/gitcoin/UserGitcoinScore'
+import { environment } from '@/config/environment'
+
+export const SufficientUserScore = ({ userScore }: UserScoreProps) => {
+  return (
+    <FormWrapper>
+      <FormHeading>Your Score: {userScore}</FormHeading>
+      <FormRow>
+        <p>
+          To place a bid your score needs to be <b>at least {environment.gitcoinRequiredScore}</b>.
```

Comment:
We have additional text "Higher score will not increase your chances in the rafffle." in desing here.